README.md

```diff
@@ -1,15 +1,418 @@
-# ID Mapping Service
+# Joint JGI / KBASE ID Mapping Service
+
+This repo contains the JGI / KBase ID Mapping Service (IMS). The service provides for mapping
+IDs in one namespace (e.g. [NCBI](https://www.ncbi.nlm.nih.gov/)) to another (e.g. JGI).
+
+For example, the `NCBI Refseq` ID `GCF_001598195.1` maps to the `KBase CI` ID `15792/22/3`,
+so the service could store the mapping from the ID `GCF_001598195.1` in the namespace
+`NCBI Refseq` to the ID `15792/22/3` in the namespace `KBase CI`, and vice versa.
 
 Build status (master):
 [![Build Status](https://travis-ci.org/jgi-kbase/IDMappingService.svg?branch=master)](https://travis-ci.org/jgi-kbase/IDMappingService)
 [![codecov](https://codecov.io/gh/jgi-kbase/IDMappingService/branch/master/graph/badge.svg)](https://codecov.io/gh/jgi-kbase/IDMappingService)
 
+## Abbreviations:
+
+* IMS - ID Mapping Service
+* JGI - Joint Genome Institute
+* CLI - Command Line Interface
+* CSL - Comma Separated List
+
+## Requirements
```

Comment:
I would put the API above all these details, especially any development or building instructions